### PR TITLE
Update NOTA page

### PR DESCRIPTION
**Pull request by zack13532**
- Make updates to no-tech-for-apartheid.md
- Updates for Ike's comments
- Updates from paper doc at 11/9 9:45 pm pt

### CODE REVIEW

File: no-tech-for-apartheid.md [hunk 1]
```diff
@@ -42,9 +42,10 @@ It is so easy to fall into despair and feel hopeless. But if we take action toge
...
+* **Mobilize**: When our leadership refuses to acknowledge how tech workers are impacted by the current crisis, we will honor our colleagues ourselves. Organizing a vigil is an opportunity for us to come together to honor the passing of tech workers like Mai. Pick a date and an entrance to your office where you will gather. To organize a vigil at your office, see [Bit.ly/VigilHowTo](https://bit.ly/VigilHowTo). 
 
-* **Learn**: Educate yourself about the current crisis. Bring your coworkers to the many virtual teach-ins that Palestinian groups have been organizing.
+* **Learn**: Educate yourself about the current crisis. Bring your coworkers to the many virtual teach-ins that Palestinian groups have been organizing. Upcoming teach-ins: 
+    * [Tech Workers Teach In (virtual)](https://us06web.zoom.us/j/86353038651?pwd=EoipHDubY5uTzFA3BIDe0du1iTAmc1.1): Tuesday 1pm PST
```
Comment (ikehz) [inline no-tech-for-apartheid.md:48]: Is "Tech Workers Teach In" a good description of this event? Seems, uh, very vague. Teach in about what? By tech workers? For tech workers? Both?

Also that link goes directly to the meeting ... should we have a registration to be able to contact people (name, email, phone, city, state, country/region, company, metro)? See http://bit.ly/NOTAMassCalls.

File: no-tech-for-apartheid.md [hunk 2]
```diff
@@ -42,9 +42,10 @@ It is so easy to fall into despair and feel hopeless. But if we take action toge
 
 * **Outreach**: Message your coworkers in your team, org, and ERGs to get them involved too, so we can build a movement that’s ready for this moment. Check out some [template communications](#template-email-to-your-coworkers-org-ergs) we have below.
 
-* **Mobilize**: When our leadership refuses to acknowledge how tech workers are impacted by the current crisis, we will honor our colleagues ourselves. Organizing a vigil is an opportunity for us to come together to honor the passing of tech workers like Mai. Pick a date and an entrance to your office where you will gather. To organize a vigil at your office, see [Bit.ly/VigilHowTo](https://bit.ly/VigilHowToo). 
+* **Mobilize**: When our leadership refuses to acknowledge how tech workers are impacted by the current crisis, we will honor our colleagues ourselves. Organizing a vigil is an opportunity for us to come together to honor the passing of tech workers like Mai. Pick a date and an entrance to your office where you will gather. To organize a vigil at your office, see [Bit.ly/VigilHowTo](https://bit.ly/VigilHowTo).
```
Comment (ikehz) [inline no-tech-for-apartheid.md:45]: This doc is not public. Perhaps we should remove the link until it is?

File: no-tech-for-apartheid.md (hunk 1 above)
Comment (zack13532) [reply no-tech-for-apartheid.md]: Update the link with your bit.ly. I think the text is pretty good as-is, though. It's a teach-in about the crisis, by Palestinian groups, for tech workers. Is there something that you think could be more clear?

File: no-tech-for-apartheid.md [hunk 3]
```diff
@@ -60,7 +61,7 @@ Our company values state [INSERT COMPANY VALUES] . Yet, our leadership has made
...
 
-I can’t stay silent like our leadership has, and I want to honor the values that our leaders won’t. Please join me in standing up against this genocide by joining the No Tech for Apartheid campaign (www.notechforapartheid.com), which is working to ensure our company and our industry is not complicit. They will be holding a mass call on [INSERT NEXT CALL] and [Bit.ly/techworkerCTA](https://bit.ly/techworkerCTA) has other actions you can take.
+I can’t stay silent like our leadership has, and I want to honor the values that our leaders won’t. Please join me in standing up against this genocide by joining the No Tech for Apartheid campaign (www.notechforapartheid.com), which is working to ensure our company and our industry is not complicit. They will be holding a mass call on [INSERT NEXT CALL] and [Bit.ly/NoTACallToAction](https://bit.ly/NoTACallToAction) has other actions you can take.
```
Comment (ikehz) [inline no-tech-for-apartheid.md:64]: Bitly is case sensitive, so this needs to be NOTACallToAction.

File: no-tech-for-apartheid.md [hunk 4]
```diff
@@ -73,6 +74,6 @@ Hi folks, I just learned that a former tech worker at Google, Mai Ubeid, and her
 
 The current death toll in Palestine is at 10,000+ and growing, with almost half of the deaths being children. Healthcare facilities are on the brink of collapse and the humanitarian situation continues to worsen.
 
-I can’t stay silent like our leadership has, and I want to honor the values that our leaders won’t. Please join me in standing up against this genocide by joining the No Tech for Apartheid campaign (www.notechforapartheid.com), which is working to ensure our company and our industry is not complicit. They will be holding a mass call on [INSERT NEXT CALL] and Bit.ly/techworkerCTA has other actions you can take.
+I can’t stay silent like our leadership has, and I want to honor the values that our leaders won’t. Please join me in standing up against this genocide by joining the No Tech for Apartheid campaign (www.notechforapartheid.com), which is working to ensure our company and our industry is not complicit. They will be holding a mass call on [INSERT NEXT CALL] and Bit.ly/NoTACallToAction has other actions you can take.
```
Comment (ikehz) [inline no-tech-for-apartheid.md:77]: Same.

File: no-tech-for-apartheid.md (hunk 1 above)
Comment (ikehz) [reply no-tech-for-apartheid.md]: I won't fret about it, but I'd like more specifics in the bullet line. If I don't read the whole "Learn" sentence, "Tech Workers Teach In" doesn't say much. Is it by or for tech workers? What is it actually about? (My understanding is this action is specifically about APEC?) & what Palestinian groups?

**Review by ikehz — COMMENTED** [review]
Needs rebase.

**Comment by shushugah** [comment]
This was already merged @ikehz correct me if I am wrong

**Pull request closed without merging**